Gemfile

```diff
@@ -162,6 +163,7 @@ end
 
 group :development do
   gem 'debugger-linecache'
+  gem 'foreman'
```

Comment:
Just noting that the author of the gem recommends against this:

https://github.com/ddollar/foreman/wiki/Don't-Bundle-Foreman

Reply:
I did notice that, but if we don't add it in the dev group then devs just have to manually install the gem. I don't see any benefit there?

Looking through the history of the repo I think the gem had a lot more dependencies in the past, and that was the reason for keeping it out of the Gemfile, but most of those dependencies have been stripped out over time and it's quite minimal now.